src/App.css

```diff
@@ -170,9 +179,81 @@
   justify-content: center;
 }
 
+.homepage-overlay {
+  background-color: beige;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+}
+
 .modal-window {
   width: 400px;
   padding: 10px;  
   border-radius: 5px;
   background-color: #ffffff;
+}
+
+.login-overlay {
+  background-color: rgb(241, 238, 238);
+}
+
+.login-window {
+  padding: 20px;  
+  border: 2px solid rgb(206, 205, 205); 
+  border-radius: 8px;
+  background-color: #ffffff;  
+}
+
+.login-textbox-email,
+.login-textbox-password,
+.login-region-submit { 
+  margin: 10px;  
+}
+
+.login-textbox-email,
+.login-textbox-password,
+.login-submit-button {  
+  padding: 12px 20px;
+  border-radius: 5px;
+  
+  font-weight: bold;
+}
+
+.login-textbox-email,
+.login-textbox-password {
+  width: 300px; 
+  border: 1px solid rgb(206, 205, 205);  
+  box-sizing: border-box;
+
+  font-size: 15px;
+  font-weight: bold;  
+}
+
+.login-submit-button {  
+  width: 100px;
+  border: 0px;
+  background-color: rgb(100, 145, 182);
+  
+  color: white;  
+}
+
+.login-link {
+  margin-left: 80px;
+  text-decoration: none;
+  font-weight: bold;
+  color: rgb(63, 97, 124);
+}
+
+input::-webkit-input-placeholder {
```

Comment:
I guess you can use `::placeholder` without any prefix

Reply:
Check it out